gridstatus/nyiso.py

```diff
@@ -625,6 +629,12 @@ def get_lmp(
         else:
             raise ValueError(f"Invalid location_type: {location_type}")
 
+        if REFERENCE_BUS_LOCATION in df["Location"].unique():
+            df.loc[
+                df["Location"] == REFERENCE_BUS_LOCATION,
+                "Location Type",
+            ] = "Reference Bus"
```

Comment:
Since the Reference Bus location is in the generator data, I vastly simplified this to just change the location type for the one Reference Bus.